### DISCUSSION TITLE
+ Symbol not being converted to space. Special Case.

### DISCUSSION THREAD

**Discussion by derpycoder** [plausible/analytics#720] [Anything Else]
Hey Guys,

Checkout this special case.

I shared my UTM param ladden URL on [Ghost's Forum](http://forum.ghost.org/). (They use discourse I believe.)

URL I shared: https://www.abhijit-kar.com/derpy-utm-builder/?utm_campaign=Derpy+Tools&utm_source=Ghost&utm_medium=Forum, which has Campaign separated by + symbol.

However the discourse changed + to %2B and now the Plausible dashboard shows campaign value wrong. See: https://www.derpycoder.com/derpy-utm-builder-a-campaign-url-builder-made-with-svelte-tailwind-snowpack/?utm_campaign=Derpy%2BTools&utm_source=Ghost&utm_medium=Forum.

How the dashboard looks:
![Screenshot from 2021-02-15 11-11-12](https://user-images.githubusercontent.com/25662120/107909662-b0efcc80-6f7e-11eb-9340-c6d988dbf9b6.png)

P.S. Plausible dashboard for my site is public. See: [Analytics for DerpyCoder](https://plausible.io/derpycoder.com).

**Comment by ukutaht** [comment]
By the time the user lands on your site, the URL already has `Derpy%2BTools` which is the percent-encoded version of `Derpy+Tools`. There's no way to know on our end that you intended to use an empty space there instead of a `+`. 

The problem is with how Discourse encodes the link you provide. You should be able to avoid the issue by changing the link in your Discourse post from `Derpy+Tools` to a more bulletproof `Derpy%20Tools`

**Reply by ukutaht** [reply]
@metmarkosaric Maybe worth changing our instructions here as well: https://plausible.io/blog/utm-tracking-tags#you-cannot-use-spaces-in-utm-tags

According to [this resource](https://community.wolfram.com/groups/-/m/t/749484), `%20` is the proper way to add whitespace in URLs. I know `+` works usually as well and I use it because it's more convenient, but sometimes it can create problems like this one with Discourse.

**Reply by metmarkosaric** [reply]
thanks. have added this now

**Reply by derpycoder** [reply]
Hehe,

I am dropping + signs from any future UTM Param link shares.
